modules/cli/src/main/scala/coursier/cli/options/SharedLaunchOptions.scala

```diff
@@ -8,21 +8,30 @@ import coursier.install.RawAppDescriptor
 // format: off
 final case class SharedLaunchOptions(
 
+  @Group(OptionGroup.launch)
   @Short("M")
   @Short("main")
     mainClass: String = "",
 
+  @Group(OptionGroup.launch)
+  @Hidden
```

Comment:
I'm shared about hiding this one. It can be convenient, even for beginners.

Reply:
Agreed it can be useful, with Spark for instance. Let's show it.